Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "cuteness"
-version = "0.9.9"
+version = "0.9.91"
```

What is this progression :sob: 

```suggestion
version = "0.9.10"
```

---

something that i am NOT good at lol